### ISSUE TITLE
System packages should not be upgraded when running bootstrap.sh

### ISSUE THREAD

**Issue by ReekenX** [EverythingMe/redash#532]
Hi! First of all - thanks for a great software.

I executed bootstrap.sh (read on the docs) and found there few issues with setup/bootstrap.sh code:

    # Base packages
    apt-get dist-upgrade
    apt-get update
    apt-get install -y python-pip python-dev nginx curl build-essential pwgen
    pip install -U setuptools

1) Why `dist-upgrade` is here at all? Users running just a bit older (by minor version) OS will need to take responsibility about upgrading tons of packages. I executed this (my OS is only one month old after previous manual upgrade) and I have received 325 packages to be upgraded.

2) If there is a really really need to have `dist-upgrade`, why it's not after `apt-get update` statement. So you forcing user to update system packages, but after `apt-get update` user might receive newer packages. That would still mean that line with `dist-upgrade` is useless.

My suggestion would be to remove whole `dist-upgrade` line. After running `apt-get update` user will get new packages for the rest of installation and dependencies will be recalculated using system package manager. That means that user will get latest packages only for re:dash.

**Comment by arikfr** [comment]
The bootstrap.sh script is meant to be run on a "clean" box, and will definitely need some tweaking to be used on an existing machine. I've added this note to the docs now (https://github.com/EverythingMe/redash/commit/1ad01d8394e08cac3920e2cde82f9acd77b71fc0).

`dist-upgrade` was added, because `apt-get install` would fail on AWS Ubuntu images without running it (or doing some other hacky workaround).
